### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by dabou**
Update .po file for french translation.

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -106,19 +128,136 @@ msgstr "Police:"
...
+msgid "<b>Bold</b>"
+msgstr "<b>Gras</b>"
+
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/HelpBox.vala:46
+msgid "**bold**"
+msgstr "**gras**"
+
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/HelpBox.vala:47
+msgid "<s>Strike</s>"
+msgstr "Souligné"
```
Comment (Philip-Scott) [inline po/fr.po]: You're missing the `<s>`s tags :)

File: po/fr.po [hunk 2]
```diff
@@ -106,19 +128,136 @@ msgstr "Police:"
...
+
+#: /home/felipe/Code/Notes-up/po/../src/Widgets/HelpBox.vala:47
+msgid "~~strike~~"
+msgstr "Barré"
```
Comment (Philip-Scott) [inline po/fr.po]: and the `~~`

File: po/fr.po (hunk 1 above)
Comment (dabou) [reply po/fr.po]: Oups :(
I will correct that

**Comment by dabou** [comment]
It seems ok now.

**Pull request merged**